contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
I think I have a compromise for the unix hier embedded in a framework with symlinks.  I will test this as soon as I get a chance.  Instead of symlinks from framework locations into the usr/{bin,lib,include,...} directories, have the symlinks go the other way where unix directories {bin,lib,include,...} are symlinks to the locations in the framework bundle.  See below for the idea:

```
Julia.framework
└── Versions
    ├── 0.5
    │   ├── Documentation
    │   │   ├── ...
    │   ├── Frameworks
    │   │   ├── libamd.dylib
    │   │   ├── libarpack.2.dylib
    │   │   ├── libarpack.dylib -> libarpack.2.dylib
    │   │   ├── ...
    │   ├── Headers
    │   │   ├── Julia.h
    │   │   └── julia
    │   │       ├── ...
    │   ├── Helpers
    │   │   └── julia
    │   ├── Julia
    │   ├── Modules
    │   │   └── module.modulemap
    │   ├── Resources
    │   │   ├── Info.plist
    │   │   ├── LICENSE.md
    │   │   ├── julia
    │   │   │   ├── base
    │   │   │   │   ├── ...
    │   ├── include -> Headers
    │   ├── bin -> Helpers
    │   ├── lib
    │   │   ├── libjulia.dylib -> ../Julia
    │   │   └── julia -> ../Frameworks
    │   ├── share -> Resources
    │   └── _CodeSignature
    │       └── CodeResources
    └── Current -> 0.5
```